src/SonarScanner.MSBuild.Common/Interfaces/IProcessRunner.cs

```diff
@@ -22,5 +22,8 @@ namespace SonarScanner.MSBuild.Common;
 
 public interface IProcessRunner
 {
+    public TextReader StandardOutput { get; }
+    public TextReader ErrorOutput { get; }
+
```

Comment:
I'm not a big fan of making `IProcessRunner` statefull. Can we instead do
```cs
bool Execute(ProcessRunnerArguments runnerArgs);
bool Execute(ProcessRunnerArguments runnerArgs, out TextReader standardOutput, TextReader errorOutput );
```

This makes testing a bit harder, but it is possbile:
https://nsubstitute.github.io/help/setting-out-and-ref-arguments/